src/core/useGLTF.tsx

```diff
@@ -3,15 +3,19 @@ import { Loader } from 'three'
 import { GLTFLoader, DRACOLoader, MeshoptDecoder } from 'three-stdlib'
 import { useLoader } from '@react-three/fiber'
 
+let dracoLoader: DRACOLoader | null = null
+
 function extensions(useDraco: boolean | string, useMeshopt: boolean, extendLoader?: (loader: GLTFLoader) => void) {
   return (loader: Loader) => {
     if (extendLoader) {
       extendLoader(loader as GLTFLoader)
     }
     if (useDraco) {
-      const dracoLoader = new DRACOLoader()
+      if (!dracoLoader) {
+        dracoLoader = new DRACOLoader()
+      }
       dracoLoader.setDecoderPath(
-        typeof useDraco === 'string' ? useDraco : 'https://www.gstatic.com/draco/versioned/decoders/1.4.0/'
+        typeof useDraco === 'string' ? useDraco : 'https://www.gstatic.com/draco/versioned/decoders/1.4.3/'
```

Comment:
just making sure, certain that 1.4.3 is ok? they broke practically everything in a recent update and i don't trust their patch updates. can we be sure this is safe?

Reply:
Great question! We were discussing this here https://github.com/google/draco/issues/751

The versioned URLs are recommended and 1.4.3 is the latest.

I would expect useGLTF to pull the latest if not overriding it with my own local version 🤷‍♂️